### PR TITLE
Update Signed Offers view in legacy UI

### PR DESCRIPTION
**Pull request by napoly**
Should solve https://github.com/haveno-dex/haveno/issues/579

### CODE REVIEW

File: desktop/src/main/java/haveno/desktop/main/offer/signedoffer/SignedOfferListItem.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+import org.apache.commons.lang3.StringUtils;
+
+import lombok.Getter;
+
+import haveno.core.offer.SignedOffer;
+import haveno.desktop.util.filtering.FilterableListItem;
+
+/**
+ * We could remove that wrapper if it is not needed for additional UI only fields.
```
Comment (woodser) [inline desktop/src/main/java/haveno/desktop/main/offer/signedoffer/SignedOfferListItem.java:28]: Comment is unclear or may be unnecessary if we're unlikely to remove the wrapper because it serves a conceptual purpose consistent with other UI classes. This class contains logic so I wouldn't think it's removable. Remove or clarify?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -14,23 +14,23 @@ jobs:
       fail-fast: false
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```
Comment (woodser) [inline .github/workflows/build.yml:17]: Why the version changes as part of this PR? I think it's ok to include but should be separated out to another commit for a total of 2 commits.

**Comment by woodser** [comment]
Please rebase to resolve conflicts.

File: desktop/src/main/java/haveno/desktop/main/offer/signedoffer/SignedOfferListItem.java (hunk 1 above)
Comment (napoly) [reply desktop/src/main/java/haveno/desktop/main/offer/signedoffer/SignedOfferListItem.java]: Seems redundant. Removed.

File: .github/workflows/build.yml (hunk 2 above)
Comment (napoly) [reply .github/workflows/build.yml:17]: Partially, yes. I had to upgrade to v3 in codeql in previous commit. This just aligns the versioning for build as well as I missed it.

**Comment by napoly** [comment]
> Please rebase to resolve conflicts.

rebased
